components/meter/meter-radial.js

```diff
@@ -0,0 +1,52 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { heading4Styles } from '../typography/styles.js';
+
+class MeterRadial extends LitElement {
+	static get styles() {
+		return [ heading4Styles, css`
+		:host {
+			display: block;
```

Comment:
I'd be curious how design sees this being used size-wize vs. the linear one. Like I can see the linear one always needing to take up as much width as its container (e.g. in a table cell)... but should the radial one? Should it maybe be `inline-block` with a default width and then they could set whatever width they want via CSS? Arguably the linear one could be similar. I really don't know what I'd prefer here.

Reply:
Bob expected the linear one to size to the container. I asked him this morning. For this one, I would expect it to be more or less constant. If it did change then the aspect ratio will stay the same.

Reply:
Yeah I agree with your thinking there. So in its case, maybe `inline-block` with a good default size and then they could modify it with `style="width:100%"` or `style="width:500px;"`. Ideally I think the CSS height is ignored and grows/shrinks relative to the width.

Reply:
Yup! Just tested it and it works very nicely. About to push commit.